### PR TITLE
chore: remove bear from mockingbird

### PR DESCRIPTION
**Pull request by ivanmjartan**
JIRA: STL-351

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
// Tiger platform
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

```
// Bear platform
extended test - cypress - integrated
extended test - cypress - isolated
extended test - cypress - record
```

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18489/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20140/ : SUCCESS in 10m 40s

File: libs/sdk-backend-mockingbird/src/recordedBackend/execution.ts [hunk 1]
```diff
@@ -486,7 +486,7 @@ function normalizedDataView(
  *
  * @param recording - recording (as obtained from the index, typically using the Scenario mapping)
  * @param dataViewId - Identifier of the data view; defaults to view with all data
- * @param resultRefType - Specify what types of refs should the backend create in the result's dimension descriptors (uri refs returned by bear, id refs returned by tiger)
+ * @param resultRefType - Specify what types of refs should the backend create in the result's dimension descriptors (id refs returned by tiger)
```
Comment (pbenes) [inline libs/sdk-backend-mockingbird/src/recordedBackend/execution.ts:489]: just wondering if we can remove the resultRefType fully, even from the functions as we call them. Also, it still has
```
   resultRefType: RecordedRefType = "uri",
```
as the default param value.

Comment (pbenes) [reply libs/sdk-backend-mockingbird/src/recordedBackend/execution.ts:489]: Separated to STL-354

**Review by pbenes — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR4834

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/4105/ : SUCCESS in 9m 25s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/1984/ : SUCCESS in 19m 33s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20143/ : SUCCESS in 10m 45s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/1774/ : SUCCESS in 11m 20s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/1761/ : SUCCESS in 10m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/5280/ : SUCCESS in 6s

**Pull request merged**